framework/db/mysql/Schema.php

```diff
@@ -53,6 +68,128 @@ class Schema extends \yii\db\Schema
         'varbinary' => self::TYPE_BINARY,
     ];
 
+    /**
+     * @inheritDoc
+     */
+    protected function resolveTableName($name)
+    {
+        $resolvedName = new TableSchema();
+        $parts = explode('.', str_replace('`', '', $name));
```

Comment:
It should not but [it doesn't seem](https://github.com/sergeymakinen/yii2/blob/3de6f8e397d336d59a7fd8c501a4132631c29563/framework/db/mysql/Schema.php#L200-L214) to bother anyone yet.
Should I fix all of this if it's possible?